tests/ruff.toml

```diff
@@ -8,8 +8,6 @@ lint.extend-ignore = [
     # This is likely to be a high-touch rule. Doing this in `tests` doesn't help circular imports.
     # Let's differ this for tests until there are auto-fixes
     "TCH001",
-    # https://beta.ruff.rs/docs/rules/#flake8-tidy-imports-tid
-    "TID251",  # banned-api - is used to prevent opaque attribute errors caused by missing optional dependencies
```

Comment:
Is the existing functionality of TID251 still enabled with the addition of the `[lint.flake8-tidy-imports.banned-api]` section?

Reply:
Nevermind - realizing now you are removing from `lint.extend-ignore` i.e. you are enabling.

Reply:
No, I'm overriding the `[lint.flake8-tidy-imports.banned-api]` rules.
To extend those rules I'd have to add them to the overridden section.
We shouldn't need to care about the compatibility imports in tests.